Shared/Interactors/GetMovieBackdropInteractor.swift

```diff
@@ -0,0 +1,21 @@
+public class GetMovieBackdropInteractor {
+
+    var presenter: GetMovieBackdropPresenter
+    var gateway: MoviesGateway
+
+    init(gateway: MoviesGateway, presenter: GetMovieBackdropPresenter) {
+        self.gateway = gateway
+        self.presenter = presenter
+    }
+
+    public func get(movie: Movie) {
```

Comment:
Usually Interactors does not have more than one implementation. So, because of this the interactor does not have a protocol.